dali/kernels/imgproc/resample/resampling_filters.cu

```diff
@@ -96,6 +98,16 @@ void InitFilters(ResamplingFilters &filters, AllocType alloc) {
 
   filters[2].rescale(6);
   filters[3].rescale(4);
+
+  if (alloc == AllocType::GPU) {
+    auto filter_data_gpu = memory::alloc_unique<float>(AllocType::GPU, total_size);
+    cudaMemcpy(filter_data_gpu.get(), filters.filter_data.get(),
+               total_size * sizeof(float), cudaMemcpyHostToDevice);
+    ptrdiff_t  diff = filter_data_gpu.get() - filters.filter_data.get();
```

Comment:
Now I wonder - maybe we should use `GetResamplingFiltersCPU` and copy everything from there?